src/commands/logstream/startStreamingLogs.ts

```diff
@@ -0,0 +1,23 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import * as vscode from 'vscode';
+import { AzureActionHandler, AzureTreeDataProvider, IAzureNode } from 'vscode-azureextensionui';
+import KuduClient from 'vscode-azurekudu';
+import { localize } from '../../localize';
+import { FunctionAppTreeItem } from '../../tree/FunctionAppTreeItem';
+import { nodeUtils } from '../../utils/nodeUtils';
+import { ILogStreamTreeItem } from './ILogStreamTreeItem';
+
+export async function startStreamingLogs(context: vscode.ExtensionContext, actionHandler: AzureActionHandler, tree: AzureTreeDataProvider, node?: IAzureNode<ILogStreamTreeItem>): Promise<void> {
+    if (!node) {
+        node = <IAzureNode<ILogStreamTreeItem>>await tree.showNodePicker(FunctionAppTreeItem.contextValue);
+    }
+
```

Comment:
I don't see a check to verify that the Streaming Log for the chosen node hasn't already been started. I don't see anything in the docs suggesting that `createOutputChannel` would perform an 'already-present' check.